pretix_eth/templates/pretix_eth/pending.html

```diff
@@ -10,9 +10,8 @@
   <p>
     <strong>
       {% blocktrans trimmed %}
-      Please pay for your ticket!  <em>DO NOT</em> change the amount (e.g. round
-      up) even if your wallet allows you to do so.  If you do, we won't be able
-      to associate your payment with your order.
+      Please send your ticket payment to the address seen below.  This address
+      is specific to your order.  <em>DO NOT</em> pay to another address.
```

Comment:
I wonder if we need to state this. Sure you need to take care of idiotic users - but IMHO stating it here hurts UX. Other services also do not state such a thing as far as I see

Reply:
Yeah, I agree actually.  Let's take it out.